README.md

```diff
@@ -0,0 +1,136 @@
+# backplane-tools
+
+backplane-tools offers an easy solution to install, remove, and upgrade a useful set of tools for interacting with OpenShift clusters, managed or otherwise.
+
+## Table of Contents
+<!-- toc -->
+- [FAQ - How do I...](#faq-how-do-i)
+  - [List available tools](#list-available-tools)
+  - [Install everything](#install-everything)
+  - [Install a specific thing](#install-a-specific-thing)
+  - [Upgrade everything](#upgrade-everything)
+  - [Upgrade a specific thing](#upgrade-a-specific-thing)
+  - [Remove everything](#remove-everything)
+  - [Remove a specific thing](#remove-a-specific-thing)
+- [Design](#design)
+  - [Directory Structure](#directory-structure)
+  - [Installing](#installing)
+  - [Upgrading](#upgrading)
+  - [Removing](#removing)
+<!-- tocstop -->
+
+## FAQ - How do I...
+Quick reference guide
+
+### List available tools
+```shell
+backplane-tools install --help
+```
+Tools available for management by backplane-tools are listed under the `Usage` section of the help output within the square brackets (`[]`).
+
+### Add the tools I've installed to my $PATH
+Add the following line to your shell's .rc file:
+```shell
+export PATH=${PATH}:${HOME}/.local/bin/backplane/latest
+```
+
+> [!IMPORTANT]
+> In order to minimize unexpected collisions and preemption, tools installed by backplane-tools are not added to your $PATH by default. In order to most effectively utilize this application, it's recommended that you add `export PATH=${PATH}:${HOME}/.local/bin/backplane` to your shell's .rc file.
```

Comment:
It seems this could be simplified by putting this like above the above section and trimmed because it's duplicating what we are saying above.
`> In order to minimize unexpected collisions and preemption, tools installed by backplane-tools are not added to your $PATH by default.`

Reply:
Good catch: edited this line to be less repetitive